app/src/main/java/org/simple/clinic/widgets/Views.kt

```diff
@@ -50,6 +51,16 @@ fun ViewGroup.hideKeyboard() {
   }
 }
 
+fun EditText.text(textToSet: CharSequence?, textWatcher: TextWatcher) {
```

Comment:
The name `text` doesn't indicate that it does the job of managing the textwatcher added to the edit text before setting the text. Can we rename the extension to better reflect this behaviour, please?

Reply:
Hmm, I thought it should be fine with the parameter names. Changing it to `setTextWithWatcher`